front/lib/api/data_sources.ts

```diff
@@ -436,144 +436,6 @@ export async function upsertDocument({
   return new Ok(upsertRes.value);
 }
 
-export interface UpsertTableArgs {
-  tableId: string;
-  name: string;
-  description: string;
-  truncate: boolean;
-  csv?: string | null;
-  tags?: string[] | null;
-  parentId?: string | null;
-  parents?: string[] | null;
-  timestamp?: number | null;
-  async: boolean;
-  dataSource: DataSourceResource;
-  auth: Authenticator;
-  title: string;
-  mimeType: string;
-  sourceUrl?: string | null;
-}
-
-export function isUpsertTableArgs(
-  args: UpsertTableArgs | UpsertDocumentArgs | undefined
-): args is UpsertTableArgs {
-  return args !== undefined && "tableId" in args;
-}
-
-export async function upsertTable({
```

Comment:
was a duplicate of `handleDataSourceTableCSVUpsert` below. Standardized on the later implementation as it had better error reporting.